src/database/queries/user/insertQuery.js

```diff
@@ -1,9 +1,8 @@
-const client = require('../../db_connection');
+const connection = require('../../db_connection');
 
-const insertQuery = (collectionName, objData) => new Promise((resolve, reject) => {
-  client.connect((error) => {
-    if (error) reject(error);
-    const database = client.db('enhance-a-smile-db');
+module.exports = (collectionName, objData) => new Promise((resolve, reject) => {
+  connection().then((dbs) => {
+    const database = dbs.production;
```

Comment:
@ahmedisam99 This isn't a required change, but it might be helpful to export a few wrappers of this function that are hard-coded to use a specific collectionName.

First you'd bring back the `const insertQuery = ...`, so your insert function is named again.  Then at the bottom of this file:
```
module.exports = {
  insertDonation: (objData) => insertQuery('donation', objData),
  insertUser: (objData) => insertQuery('user', objData),
  insertAdmin: (objData) => insertQuery('admin', objData),
```
.. or whatever all of your different collectionNames are.  This might seem pointless at first :) but the idea is that you are protecting yourself against typos in the collectionName in the future as you use these functions.

Reply:
@tomduggan85,
nice idea :+1: 
I'm gonna use it